src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -332,7 +332,7 @@ object DeltaConfigs extends DeltaLogging {
     Some(Protocol(0, 2)))
 
   /**
-   * Whether this table will automagically optimize the layout of files during writes.
+   * Whether this table will automatically optimize the layout of files during writes.
```

Comment:
not a typo :P auto-magically is a cute term instead of automatically

Reply:
haha, let's keep it magically.